### DISCUSSION TITLE
Is it possible to view previous revisions of my data?

### DISCUSSION THREAD

**Discussion by AlexandriaLittle** [Budibase/budibase#5883] [Help]
Hi All,

I'm looking into making another new app for my company. This one is for a new purchasing workflow. Because it ties in so closely with our financials we're looking to have strict control over access and the ability to audit our records. Is there some way for me to display previous versions of data saved in the Budibase DB? If there isn't then I'll have to build an event log into the app as well which is something I'd hope to avoid if possible.

Many thanks,
Alexa Little

**Comment by melohagan** [comment]
Hi @AlexNCurrie 

There is a brief discussion of this here: https://github.com/Budibase/budibase/discussions/3190

I don't see anything on the roadmap for this, although it may be part of the larger RBAC epic: https://github.com/Budibase/budibase/discussions/965
**Q3**: https://github.com/Budibase/budibase/projects/10
